dispatches/models/renewables_case/submit_job.py

```diff
@@ -0,0 +1,72 @@
+import os
+
+this_file_path = os.path.dirname(os.path.realpath(__file__))
+
+
+def submit_job(
```

Comment:
Does this file really belong in DISPATCHES?  This seems very specific to a single developer / platform / use case.

Reply:
@xiangao1 Are you fine with removing this from the branch?